README.md

```diff
@@ -80,9 +85,10 @@ Here is a list of the main folders/files of the repository.
 ```
 iscsc.fr
 ├── backend                 *contains the server-side code and API* 
-│   ├── controllers/         *usefull js function for each model*
-│   ├── models/              *contains the database models*
-│   ├── routes/              *routes and functions to executes for each model*
+│   ├── controllers         *usefull js functions for each model*
```

Comment:
you removed the "/" here :thinking:

Reply:
Indeed thanks!
cf [0988259](https://github.com/iScsc/iscsc.fr/pull/2/commits/0988259c1be4406c690dece3bccf240a988f4561)

Reply:
thanks :yum: